src/components/app/ProfileViewer.css

```diff
@@ -11,4 +11,31 @@
   background-color: #fff;
   animation-name: profileViewerFadeIn;
   animation-duration: 200ms;
-}
\ No newline at end of file
+}
+
+.profileViewerName {
+  height: 24px;
+  padding: 0 6px 0 2px;
+  border-right: 1px solid var(--grey-30);
+  line-height: 24px;
+  font-weight: bold;
+  color: var(--grey-60);
+}
+
+.profileViewerZipButton {
+  width: 18px;
+  height: 18px;
+
+  margin: 3px;
+  padding: 0;
+  border: 1px solid var(--green-60);
+
+  color: #000;
+  background-color: var(--green-50);
+  border-radius: 3px;
+
```

Comment:
nit: there's an empty line that shouldn't be here

Reply:
Oh yesssss, I filed a prettier in CSS follow-up: #878